researcher_ids/RI_Claims_V1.md

```diff
@@ -60,200 +76,298 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-
-    5.  The Passport for the /userinfo request MUST contain "ga4gh" as a space-
-        separated sub-string within the "scope" claim in
-        order to include the "ga4gh" OIDC claim as part of the response.
-
-    6.  The Identity Broker MUST NOT include claims from upstream Identity
-        Brokers as part of the "ga4gh" OIDC claim unless the Identity Broker
-        wishes to follow an implicit trust security model and take
-        responsibility for presenting those upstream claims to a specific
-        audience. For example, an organization may provide an
-        intraorganizational security service that filters and flattens all
-        claims that meet that organization’s security and trust model on behalf
-        of many Claims Clearinghouses within that organization where each such
-        client of this service places absolute trust in its ability to filter
-        claims into a flattened claim set that gets signed as a Passport.
-
-8.  <a name="requirement-8"></a> RI Claims are designed for machine
+    -   The same document can encode Passport Visas for any
+        [Passport Clearinghouse](#passport-clearinghouse) to evaluate when
+        requesting pre-authorization for a longer duration (e.g. a request can
+        establish intent to access a resource over the next 60 days, even if
+        this access ends up being revoked after 15 days for other reasons)
+        without the creator of the document requiring knowledge of the policies
+        of the Passport Clearinghouse that inspects the Passport Visas.
+
+4.  <a name="requirement-4"></a> Passport Visas MUST have an indication of
+    which organization asserted the claim (i.e. the "[source](#source)" field),
+    but Passport Visas do not generally indicate individual persons involved
+    in making the assertion (i.e. who assigned/signed the claim) as detailed
+    identity information is not needed to make an access decision.
+
+5.  <a name="requirement-5"></a> Additional information about identity
+    that is not needed to make an access decision SHOULD not be included in the
+    Passport Visas. Identity description, encoding audit details, other data
+    for non-access purposes are not the intent of these Passport Visas,
+    and must be handled via other means beyond the scope of this specification
+    should they be needed for entities and systems with sufficient authority to
+    process such information.
+
+6.  <a name="requirement-6"></a> All Passport Visas within the
+    `ga4gh_passport_v1` scope eligible for release to the requestor MUST be
+    consistently provided. Reasons for limiting exchange may include user
+    approval, contractual limitations, regulatory restrictions, or filtering
+    claims to only the subset needed for a particular purpose, etc.
+
+7.  <a name="requirement-7"></a> The [Passport JWT Claim](#passport-jwt-claim)
+    is only included in the Passport if the `ga4gh_passport_v1` scope is
```

Comment:
It would be elegant to make this spec focus on RI claims and leave all protocol stuff ("scope") to the AAI spec.

Reply:
Required to specify the name of the passport scope as the AAI spec leaves this as an abstraction to handle more than one spec using it.